src/internal/m365/service/sharepoint/export.go

```diff
@@ -0,0 +1,57 @@
+package sharepoint
+
+import (
+	"context"
+
+	"github.com/alcionai/clues"
+
+	"github.com/alcionai/corso/src/internal/common/idname"
+	"github.com/alcionai/corso/src/internal/data"
+	"github.com/alcionai/corso/src/internal/m365/collection/drive"
+	"github.com/alcionai/corso/src/pkg/backup/details"
+	"github.com/alcionai/corso/src/pkg/control"
+	"github.com/alcionai/corso/src/pkg/export"
+	"github.com/alcionai/corso/src/pkg/fault"
+	"github.com/alcionai/corso/src/pkg/logger"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+// ProduceExportCollections will create the export collections for the
+// given restore collections.
+func ProduceExportCollections(
+	ctx context.Context,
+	backupVersion int,
```

Comment:
Any chance of getting an `ExportProducerConfig` similar to the [backup and restore versions](https://github.com/alcionai/corso/blob/main/src/internal/operations/inject/containers.go#L24) we have?  As a separate PR, of course.

Reply:
Hmm, that could be useful to unify the interface. I'll add it my backlog.